src/pudl/workspace/setup.py

```diff
@@ -67,7 +67,9 @@ def get_defaults():
     # Ensure that no matter what the user has put in this file, we get fully
     # specified absolute paths out when we read it:
     pudl_in = pathlib.Path(default_workspace["pudl_in"]).expanduser().resolve()
-    pudl_out = pathlib.Path(default_workspace["pudl_out"]).expanduser().resolve()
+    pudl_out = (
```

Comment:
This combines the output directories instead of having `sqlite` and `parquet` dirs. That does mean people with an existing workspace will need to add an `output` directory at the top level of their workspace.

Reply:
So we'll have `workspace/output/sqlite` and `workspace/output/parquet`?

Reply:
No we'll just have a single `workspace/output` directory, and the `sqlite_dir` and `parquet_dir` params will point there